### ISSUE TITLE
client.get_buckets() doesn't seem to work

### ISSUE THREAD

**Issue by bkircher** [gridscale/gridscale_api_client_python#2]
Steps to reproduce:

Get an account with no buckets in object storage. curl the public API and verify:
```
$ GS_USER_ID=xyz; GS_API_TOKEN=xyz; curl https://api.gridscale.cloud/objects/objectstorages/buckets -H "X-Auth-UserID: $GS_USER_ID" -H "X-Auth-Token: $GS_API_TOKEN"
{"buckets": []}
```
Run test script (see below):
```
$ GS_USER_ID=xyz GS_API_TOKEN=xyz python test.py{'buckets': None}
```

So far so good. Now create a bucket in the panel, let say 'fubar' and curl again.
```
$ GS_USER_ID=xyz; GS_API_TOKEN=xyz; curl https://api.gridscale.cloud/objects/objectstorages/buckets -H "X-Auth-UserID: $GS_USER_ID" -H "X-Auth-Token: $GS_API_TOKEN"
{"buckets": [{"name": "fubar", "usage": {"num_objects": 0, "size_kb": 0}}]}
```
Looks good.

Now the API client:
```
$ GS_USER_ID=xyz GS_API_TOKEN=xyz python test.py
{'buckets': None}
```
Looks not okay.

If you add more buckets, get_buckets keeps returning none.

Test script (test.py):
```python
import os
from pprint import pprint

from gs_api_client import Configuration, SyncGridscaleApiClient


user_id = os.getenv("GS_USER_ID")
assert user_id

api_token = os.getenv("GS_API_TOKEN")
assert api_token


config = Configuration()
config.host = "https://api.gridscale.cloud"
config.api_key["X-Auth-Token"] = api_token
config.api_key["X-Auth-UserId"] = user_id
config.debug = False

client = SyncGridscaleApiClient(configuration=config)
pprint(client.get_buckets())
```

**Comment by Aldemuro** [comment]
The fix for this BUG has been implemented and will be available in next version.

**Comment by Aldemuro** [comment]
This is fixed in version 1.1.5 https://pypi.org/project/gs-api-client
